windows/server_stats.ps1

```diff
@@ -1,32 +1,57 @@
 # Note, all code to be executed on the remote server needs to belong
 # in the ServerStats code block
 $ServerStats = {
```

Comment:
Having this parameter access here is really nice, I like that addition of documentation